core/src/test/java/com/google/cloud/sql/core/InternalConnectorRegistryTest.java

```diff
@@ -242,13 +244,45 @@ public void create_notAuthorized() throws IOException {
               .withCloudSqlInstance("myProject:myRegion:NotMyInstance")
               .withIpTypes("PRIMARY")
               .build());
-      fail();
+      fail("Expected TerminalException");
+    } catch (TerminalException e) {
```

Comment:
Would `assertThrows` be a good fit here?

Reply:
Updated. Thanks for the suggestion.